hw/top_englishbreakfast/data/autogen/top_englishbreakfast.gen.hjson

```diff
@@ -502,6 +502,10 @@
           domain: "0"
         }
       }
+      alert_connections:
+      {
+        fatal_fault: null
```

Comment:
Should we populate the default connection if no explicit connection is provided in the hjson?